docs/user-manuals/resourcedistribution.md

```diff
@@ -0,0 +1,223 @@
+---
+title: ResourceDistribution
+---
+
+For the scenario, where the namespace-scoped resources such as Secret and ConfigMap need to be distributed or synchronized to different namespaces, the native k8s currently only supports manual distribution and synchronization by users one-by-one, which is very inconvenient. 
+
+Typical example: when users need to use the imagePullSecrets capability of SidecarSet, they must repeatedly create corresponding Secrets in relevant namespaces, and ensure the correctness and consistency of these Secret configurations.
+
+Therefore, in the face of these scenarios that require the resource distribution and synchronization **across namespaces**, we provide a convenient tool, namely **ResourceDistribution**, to do this automatically. 
```

Comment:
plz emphasize that we **continuously synchronization** configmap&secrets

Reply:
done

Reply:
done